src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/DraftApprenticeship/DraftApprenticeshipAddEmailRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace SFA.DAS.ProviderCommitments.Web.Models.DraftApprenticeship
+{
+    public class DraftApprenticeshipAddEmailRequest: DraftApprenticeshipRequest
+    {      
+      
+    }
+}
```

Comment:
Is this class used, it's very unusual to have a class which implements no additional properties or methods